### ISSUE TITLE
this repository does not seem to compile might you be interested in helping cleanup a readme?

### ISSUE THREAD

**Issue by carlkeifer3** [undisbeliever/untech-editor#3]
just trying to build this whole project, but without instructions it doesn't seem to want to build. it's also firmly in gnumake which has trouble outside certain linux flavors. any chance you could help?

**Comment by undisbeliever** [comment]
Certainly.  This was project developed and tested on ArchLinux.  It's been a few years since I compiled it on other systems.  

If you could tell me which system/compiler you use, I can try to compile untech-editor for it and write some improved build instructions.  However, I do not have access of a OSX machine and will not be able to test on OSX.

I cannot remember what GNUmake features I'm using.

Over the week, I'm spin up an OpenBSD VM and try to craft a makefile that works with OpenBSD make and GNU make.  Hopefully that will improve the portability of the project.

<br/>

Please be aware that untech-engine and untech-editor are currently low-priority projects for me (I'm mostly focusing on my [unnamed-snes-game](https://github.com/undisbeliever/unnamed-snes-engine) at the moment).

**Comment by undisbeliever** [comment]
Looking through [A Tutorial on Portable Makefiles](https://nullprogram.com/blog/2017/08/20/) and the OpenBSD make manpage, I doubt I would be able to port my GNUmakefile to BSD make.

I think I will switch the build system to CMake instead.
